substrate/frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -834,7 +834,7 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 			.storages
 			.iter()
 			.filter_map(|storage| {
-				if storage.cfg_attrs.is_empty() {
```

Comment:
@kianenigma do you remember why we check `if storage.cfg_attrs.is_empty()` here?

Reply:
It seems it's to avoid this compile error https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5317738

I wonder if there's a better way to handle that.